synapse/federation/sender/__init__.py

```diff
@@ -358,8 +358,10 @@ async def get_destinations_for_event(
                 async def get_federatable_events_and_destinations(
                     events: Iterable[EventBase],
                 ) -> List[Tuple[EventBase, Collection[str]]]:
-                    with Measure(self.clock, "fetch_destinations_for_events"):
-                        # Get destinations for events, skip if get_destinations_for_event returns None
+                    with Measure(self.clock, "get_destinations_for_events"):
+                        # Fetch federation destinations per event,
+                        # skip if get_destinations_for_event returns None,
```

Comment:
this isn't true, is it? c'mon, it shouldn't be my job to point this out.

Reply:
:facepalm: 

You're now legally allowed to slap me if i apply review suggestions before i've had coffee, will fix :+1: